Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG JENKINS_SLAVE
+FROM jenkins/jnlp-slave:${JENKINS_SLAVE}
+USER root
+
+ARG PYTHON_MAJOR
+ARG PYTHON_PATCH
+RUN export PYTHON_SEMVER="${PYTHON_MAJOR}.${PYTHON_PATCH}" && \
+  apt-get update && \
```

Comment:
FYI: For convention:

1. You put `|`, `&&` or `;` at the start of the line
Such as like this:
```
RUN set -eux \
	&& rm -rf /docker-entrypoint.d || true \
	&& mkdir -p /etc/httpd-custom.d \
    ;  echo "test" \
    | grep
```
This way it is easier to see if you are using a "all must pass", "only one must pass" or distinguish between pipes